sys-boot/os-prober/os-prober-1.79-r2.ebuild

```diff
@@ -0,0 +1,105 @@
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
repoman complains about the header.  Just change the date from `1999-2021` to `1999-2022`.

Reply:
Thanks Peter, all fixed